services/todo-legacy/repositories/todo.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  AnyObject,
+  DataSourceConstructor,
+  DefaultCrudRepository,
+  Filter,
+  Model,
+  ModelBaseConstructor,
+  ObjectType,
+  Repository,
+  Where,
+  juggler
+} from '@loopback/repository';
+import { inject, Application } from '@loopback/core';
+
+export class TodoRepository {
+  // FIXME(kev): The juggler.ModelBase type doesn't carry any definitions
+  // for the CRUD functions that typically get mixed in when createModel
+  // makes a definition, so we have to use type "any" to avoid complaint!
+  model: juggler.ModelBase | any;
```

Comment:
See https://github.com/strongloop/loopback-next/blob/master/packages/repository/src/loopback-datasource-juggler.ts#L270. We should use `juggler.PersistedModel` instead.